src/karmen_backend/server/services/mailer/templates/assets/license.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
It's not clear what the licence applies to. It would be IMHO clearer to move branded template files to a separate folder.

Reply:
It applies to templates in this very folder. I took the base from the referenced repository and just slightly modified it. But it's MIT licensed so I pasted the license there just to pay the guy  his credits.